### PR TITLE
Basic Navigation structure done in .tsx

### PR DESCRIPTION
**Pull request by waynew99**
Basic Navigation structure done in .tsx

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      <BottomTabNav />
+      <StatusBar style="light" />
+
+    </NavigationContainer>
+
+  );
+}
+
+
+const styles = StyleSheet.create({
```
Comment (Nicholas-Sliter) [inline App.tsx:21]: these should be removed

**Review by Nicholas-Sliter — CHANGES_REQUESTED** [review]

**Comment by Nicholas-Sliter** [comment]
Looks great overall!

File: App.tsx (hunk 1 above)
Comment (waynew99) [reply App.tsx]: Thx. Removed!

**Review by Nicholas-Sliter — APPROVED** [review]

**Review by Cliffy2000 — APPROVED** [review]
Nice!

**Pull request merged**